tests/jobs/design_kw/test_design_kw.py

```diff
@@ -190,6 +190,24 @@ def test_is_perl():
     assert not design_kw.is_perl(file_name, template)
 
 
+def test_is_xml():
```

Comment:
This only unit-tests the `is_xml` function I see, there should perhaps be some asserts on  `all_matched(...) == True`

Reply:
Added xml format files as template files in test

Reply:
I have added a test that run a case using template model file  where the model file is in xml format.